manifests/python.yml

```diff
@@ -453,6 +453,8 @@ tests/:
       TestDynamicConfigV1: missing_feature
       TestDynamicConfigV2: missing_feature
     test_span_links.py: missing_feature
+    test_telemetry.py:
+      Test_First_Trace_Telemetry: missing_feature
```

Comment:
For my own understanding, is this a missing feature in Python only because your Python tracer library changes are not merged yet?

Reply:
correct, I am checking with the python team to see if we can include it in the 2.4.0 release. If so, I will update the version here